### ISSUE TITLE
Error in base::rowSums(meth_mat[, case_idx]) :  'x' must be an array of at least two dimensions

### ISSUE THREAD

**Issue by powerhorse1986** [sartorlab/methylSig#51]
Hi,

I am trying to do DMR on one of our test dataset which has one sample for case and two samples for control. While I was running the diff_methylsig function, I encountered such an error "Error in base::rowSums(meth_mat[, case_idx]) :  'x' must be an array of at least two dimensions".

After diving into the source code, I realized that in the meth_mat, there's only one column of data for case. May I know if methySig requires a minimum number of samples? For example, greater than one?

Thanks.

**Comment by sartorma** [comment]
Yes, methylsig requires at least two replicates to test for differential methylation.

**Comment by powerhorse1986** [comment]
Thank you so much for the information.
